frontend/src/pages/create_campaign/Campaign/index.js

```diff
@@ -0,0 +1,158 @@
+import React from "react";
+import { FormControlLabel, Switch } from "@mui/material";
+import PropTypes from "prop-types";
+import AdapterDateFns from "@mui/lab/AdapterDateFns";
+import LocalizationProvider from "@mui/lab/LocalizationProvider";
+import DateTimePicker from "@mui/lab/DateTimePicker";
+import Dropzone from "react-dropzone";
+import {
+  CampaignContainer,
+  CampaignDropzone,
+  CampaignRowDiv,
+  SwitchRowDiv,
+  CampaignTextField,
+} from "./campaignTab.styled";
+import { fileToDataUrl } from "../../../utils";
+
+const CampaignTab = ({ isSelected, campaignData }) => {
+  const {
+    campaignName,
+    setCampaignName,
+    startDate,
+    setStartDate,
+    endDate,
+    setEndDate,
+    description,
+    setDescription,
+    interviewStage,
+    setInterviewStage,
+    scoringStage,
+    setScoringStage,
+    cover,
+    setCover,
+  } = campaignData;
+  console.log(campaignData);
+  const onFileUpload = async (acceptedFiles) => {
+    const fileUrl = await fileToDataUrl(acceptedFiles[0]);
+    setCover(fileUrl);
+  };
+
+  return (
+    isSelected && (
+      <CampaignContainer>
+        <Dropzone
+          onDrop={(acceptedFiles) => onFileUpload(acceptedFiles)}
+          accept={["image/jpeg", "image/jpg", "image/png", "image/gif"]}
+          minSize={1024}
+          maxSize={3072000}
+        >
+          {({ getRootProps, getInputProps }) => (
+            <section>
+              <CampaignDropzone {...getRootProps()}>
+                <input {...getInputProps()} />
+                {cover === null && (
+                  <p>
+                    Drag and drop your campaign cover image, or click to select
+                    an image
+                  </p>
+                )}
+                {cover !== null && <img src={cover} alt="campaign cover" />}
+              </CampaignDropzone>
+            </section>
+          )}
+        </Dropzone>
+        <CampaignTextField
+          label="Campaign Name"
+          variant="outlined"
+          value={campaignName}
+          onChange={(e) => setCampaignName(e.target.value)}
+        />
+        <CampaignTextField
+          label="Campaign Description"
+          variant="outlined"
+          multiline
+          rows={10}
```

Comment:
could we have the rows dependent on the number of words/chars expected in the question? e.g. if 100 words expected we have 3 rows, but if 300 words expected we push it to 9. That way at a glance its easy to see which questions require more effort

Reply:
wait LMAO this is a description textbox hahah pls ignore

but i guess keep the idea around for application page ^__^